app/jobs/reactions/create_job.rb

```diff
@@ -0,0 +1,19 @@
+module Reactions
+  class CreateJob < ApplicationJob
+    queue_as :reaction_create
+
+    def perform(user_id:, reactable_id:, reactable_type:, category:)
+      user = User.find_by(id: user_id)
+      reactable = reactable_type.constantize.find_by(id: reactable_id)
+
+      return unless user && reactable
+
+      Reaction.create(
```

Comment:
`create` will fail silently if something is wrong. If you use `create!` instead of it, an exception will be raised and we'll notice the error.

Reply:
I did not know that! Thank you for catching this. I'm going to fix it.